Dockerfile

```diff
@@ -63,21 +64,30 @@ RUN sudo apt-get install --yes nodejs
 RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh
 
 # Erlang + Elixir, might not need the apt-get update here?
-RUN wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb \
-    && dpkg -i erlang-solutions_1.0_all.deb \
-    && rm -f erlang-solutions_1.0_all.deb \
-    && apt-key update \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends esl-erlang elixir \
-    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+# Erlang has serious issues with installation
+#RUN wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb \
```

Comment:
Breaks horribly because these can't be installed from Xenial repos:

libssl0.9.8 libwxbase2.8-0 libwxgtk2.8-0

Also does not detect the presence of erlang when trying to install Elixir, sigh.

LibSSL is a builtin, no idea why it needs a pre 1.x version.